Documentation/Events/Events/Core/Mail/AfterMailerSentMessageEvent.rst

```diff
@@ -0,0 +1,63 @@
+.. include:: /Includes.rst.txt
+.. index:: Events; AfterMailerSentMessageEvent
+.. _AfterMailerSentMessageEvent:
+
+===========================
+AfterMailerSentMessageEvent
+===========================
+
+.. versionadded:: 12.0
+
+This event is dispatched as soon as the message has been sent via the
+corresponding :php:`Symfony\Component\Mailer\Transport\TransportInterface`.
+It receives the current mailer instance, which depends on the implementation -
+usually :php:`TYPO3\CMS\Core\Mail\Mailer`. It contains the
+:php:`Symfony\Component\Mailer\SentMessage` object, which can be retrieved
+using the :php:`getSentMessage()` method.
+
+Example
+=======
+
+Registration of the event listener:
+
+.. code-block:: yaml
+   :caption: EXT:your_extension/Configuration/Services.yaml
```

Comment:
```suggestion
   :caption: EXT:my_extension/Configuration/Services.yaml
```

Lets use the same extension name all over: out default is my_extension and Vendor\MyExtension